docs/source/deployment/aws_ecs.rst

```diff
@@ -86,32 +89,58 @@ Use the IrisClassifier BentoService from the :doc:`Quick start guide<../quicksta
     }
 
 
-==================================================================
-Build and push docker image to AWS ECR(Elastic Container Registry)
-==================================================================
+The BentoML saved bundle created can now be used to start a REST API Server hosting the
+BentoService and available for sending test request:
+
+.. code-block:: bash
+
+    # Start BentoML API server:
+    bentoml serve IrisClassifier:latest
+
+
+.. code-block:: bash
+
+    # Send test request:
+    curl -i \
+      --header "Content-Type: application/json" \
+      --request POST \
+      --data '[[5.1, 3.5, 1.4, 0.2]]' \
+      http://localhost:5000/predict
+
+======================================================================
+Dockerize BentoML model server for AWS ECR(Elastic Container Registry)
```

Comment:
Title: Dockerize BentoML model server for deployment

explain what is AWS ECR and why it's needed in the content below